proto/stride/stakeibc/validator.proto

```diff
@@ -16,7 +16,15 @@ message ValidatorExchangeRate {
 message Validator {
   string name = 1;
   string address = 2 [ (cosmos_proto.scalar) = "cosmos.AddressString" ];
-  string delegation_amt = 5 [
+  string balanced_delegation = 5 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.nullable) = false
+  ];
+  string unbalanced_delegation = 8 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.nullable) = false
+  ];
+  string cumulative_lsm_intake = 9 [
```

Comment:
This number only goes up.  But why do we need it?  Over time it could become larger than the total amount of ATOM which exists if people were to loop for some unlikely reason.  This is unrelated to any global limits on amount of LSM possible at any given time.

Reply:
it's only there for the slashing safety mechanism. One other option I considered is we can reset it every time we get a response from the query. I'd be in favor of this approach, but we'd need a good variable name otherwise I could see it causing more confusion

maybe:
`progress_towards_exchange_rate_query`
?

Reply:
Sure sounds reasonable and makes it clearer where it will be used.